demos/using-dev-build/vite.plugins.ts

```diff
@@ -0,0 +1,38 @@
+import { readFileSync } from "fs";
+import { join } from "path";
+
+export const readCanisterIds = ({
+  prefix,
+  network,
```

Comment:
why not read the network here?

Reply:
because it is also needed elsewhere currently in `vite.config.ts` that uses also `vite.plugin.ts` - avoid code duplication